console/src/components/IconButton/IconButton.js

```diff
@@ -0,0 +1,56 @@
+import AgateIcon from '@enact/agate/Icon';
+import Button from '@enact/agate/Button';
+import kind from '@enact/core/kind';
+import React from 'react';
+
+import componentCss from './IconButton.less';
+
+const Icon = kind({
+	name: 'Icon',
+	styles: {
+		css: componentCss,
+		className: 'icon'
+	},
+	computed: {
+		className: ({css, smallest, styler}) => {
+			return styler.append(smallest ? css.smallest : '');
+		}
+	},
+	render: (props) => {
+		delete props.smallest;
+		return (
+			<AgateIcon {...props} />
+		);
+	}
+});
+
+const IconButton = kind({
+	name: 'IconButton',
+	styles: {
+		css: componentCss
+	},
+	computed: {
+		className: ({css, smallest, styler}) => {
+			return styler.append(smallest ? css.smallest : '');
+		}
+	},
+	render: ({children, css, icon, smallest, ...rest}) => {
+		return (
+			<Button
+				css={css}
+				icon={
+					<Icon
+						small={rest.small}
```

Comment:
Rather than pass `rest` values, could we destructure `small` from the props above and explicitly pass that to `Icon` and `Button`?